en/docs/reference/mediators/about-mediators.md

```diff
@@ -38,26 +38,70 @@ Mediators are classified as follows based on whether or not they access the mess
-
-**Filter Mediators**
-
-[Filter]({{base_path}}/reference/mediators/filter-mediator) | [Validate]({{base_path}}/reference/mediators/validate-mediator) | [Switch]({{base_path}}/reference/mediators/switch-mediator) | 
-
-**Transform Mediators**
-
-[XSLT]({{base_path}}/reference/mediators/xslt-mediator) | [FastXSLT]({{base_path}}/reference/mediators/fastxslt-mediator) | [URLRewrite]({{base_path}}/reference/mediators/urlrewrite-mediator) | [XQuery]({{base_path}}/reference/mediators/xquery-mediator) | [Header]({{base_path}}/reference/mediators/header-mediator) | [Fault]({{base_path}}/reference/mediators/fault-mediator) | [PayloadFactory]({{base_path}}/reference/mediators/payloadfactory-mediator) | [JSONTransform]({{base_path}}/reference/mediators/json-transform-mediator) |
-
-**Advanced Mediators**
-
-[Cache]({{base_path}}/reference/mediators/cache-mediator) | [ForEach]({{base_path}}/reference/mediators/foreach-mediator) | [Clone]({{base_path}}/reference/mediators/clone-mediator) | [Store]({{base_path}}/reference/mediators/store-mediator) | [Iterate]({{base_path}}/reference/mediators/iterate-mediator) | [Aggregate]({{base_path}}/reference/mediators/aggregate-mediator) | [Callout]({{base_path}}/reference/mediators/callout-mediator) | [Transaction]({{base_path}}/reference/mediators/transaction-mediator) | [Throttle]({{base_path}}/reference/mediators/throttle-mediator) | [DBReport]({{base_path}}/reference/mediators/db-report-mediator) | [DBLookup]({{base_path}}/reference/mediators/dblookup-mediator) | [EJB]({{base_path}}/reference/mediators/ejb-mediator) | [Binder]({{base_path}}/reference/mediators/builder-mediator) | [Entitlement]({{base_path}}/reference/mediators/call-mediator) | [OAuth]({{base_path}}/reference/mediators/call-mediator) | [Smooks]({{base_path}}/reference/mediators/smooks-mediator) | [Data Mapper]({{base_path}}/reference/mediators/data-mapper-mediator) | 
-
-**Extension Mediators**
-
-[Class]({{base_path}}/reference/mediators/class-mediator) | [Script]({{base_path}}/reference/mediators/script-mediator) |
+## List of mediators
+
+WSO2 Micro Integrator includes a comprehensive library of mediators that provide functionality for implementing widely used [Enterprise Integration Patterns (EIPs)]({{base_path}}/learn/enterprise-integration-patterns/eip-overview/). You can also easily write a custom mediator to provide additional functionality using various technologies such as Java, scripting, and Spring.
```

Comment:
Let's not mention about Spring. We have deprecated this mediator (Spring Mediator) and all the relevant docs are removed.

Reply:
Removed via https://github.com/wso2/docs-mi/pull/852/commits/a5ee30bc2f229b92138d9359522181bf80c788dd.